palm/plugins/dbt/dbt_containerizer.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+from typing import Optional
+from palm.containerizer import PythonContainerizer
+import sys
+from palm.palm_exceptions import AbortPalm
+import click
+
+class DbtContainerizer(PythonContainerizer):
+    """
+    Containerizer for dbt projects.
+    """
+    def __init__(self, ctx, template_dir: Path, dbt_version: Optional[str] = '0.21.0') -> None:
+        self.ctx = ctx
+        self.project_name = ctx.obj.palm.image_name
+        self.template_dir = template_dir
+        self.dbt_version = dbt_version
+        self.package_manager = ''
+        self.profiles_file = ''
+    
+    def run(self) -> None:
+        """Runs the containerizer.
+
+        Returns:
+            None
+        """
+        super().check_setup()
+        self.package_manager = super().detect_package_manager()
+        self.profiles_file = self.detect_profiles_file()
+        super().generate(self.target_dir, self.replacements)
+        
+
+    def detect_profiles_file(self) -> str:
+        """Determines whether or not there is a profiles.yml file in the project root.
+
+        Returns:
+            str: exists | unknown
+        """
+        if self.has_profiles_file():
+            return "exists"
```

Comment:
Are we using this string somewhere? If not I think we can just `return` and remove the type hint on line 32

Reply:
This is related to the discussion point I hoped to bring up about whether or not (and how) we wanted to handle the yaml v yml situation. I'll remove it and then we can add it back later if it becomes an issue we want to handle.

Reply:
I think we are handling both already?
```
        profiles_files = ['profiles.yml', 'profiles.yaml']

        for file in profiles_files:
```

Reply:
Yes, I did at least do that much. I'll explain about profiles.yml validation in co-working